src/delivery-options/components/RecursiveForm/RecursiveForm.vue

```diff
@@ -87,12 +87,14 @@
             v-text="choice.plainLabel || $configBus.strings[choice.label]" />
           <span
             v-if="!!choice.priceTag && !isSelected(choice)"
+            v-test="'priceTag'"
```

Comment:
is het nodig om deze in single quotes te zetten?

Reply:
yes, v-test is een directive en daarbinnen is het altijd javascript